{{cookiecutter.repostory_name}}/app/Dockerfile

```diff
@@ -23,6 +23,7 @@ RUN pip3 install --user --no-cache-dir -r requirements.txt
 COPY ./src/ /root/src/
 
 RUN python3 -m compileall -b -f -q /root/
+RUN SECRET_KEY=0 CSP_ENABLED= DATABASE_URL= python3 manage.py collectstatic --no-input --clear
```

Comment:
By tricking with these env variables we can run `collectstatic` during the container build

Reply:
sure, but please add it as a comment